lua/astrocommunity/diagnostics/lsp_lines-nvim/init.lua

```diff
@@ -1 +1,17 @@
-return { "https://git.sr.ht/~whynothugo/lsp_lines.nvim", event = "LspAttach", opts = {} }
+return {
+  "https://git.sr.ht/~whynothugo/lsp_lines.nvim",
+  event = "LspAttach",
+  keys = {
+    {
+      "<Leader>uD",
+      function()
+        vim.diagnostic.config {
+          virtual_text = not require("lsp_lines").toggle(),
+        }
+      end,
+      desc = "Toggle virtual diagnostic lines",
+    },
+  },
+  init = function() require("astronvim.utils.ui").toggle_diagnostics() end,
+  opts = {},
```

Comment:
```suggestion
  opts = {},
  config = function(_, opts)
    -- disable diagnostic virtual text
    local lsp_utils = require "astronvim.utils.lsp"
    lsp_utils.diagnostics[3].virtual_text = false
    vim.diagnostic.config(lsp_utils.diagnostics[vim.g.diagnostics_mode])
    require("lsp_lines").setup(opts)
  end,
```

Reply:
Will test this tomorrow as soon as I can